root/report/DuplicateArtists.js

```diff
@@ -0,0 +1,111 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import {withCatalystContext} from '../context';
+import Layout from '../layout';
+import formatUserDate from '../utility/formatUserDate';
+import {l} from '../static/scripts/common/i18n';
+import PaginatedResults from '../components/PaginatedResults';
+import loopParity from '../utility/loopParity';
+import EntityLink from '../static/scripts/common/components/EntityLink';
+import FormRow from '../components/FormRow';
+import FormSubmit from '../components/FormSubmit';
+
+import FilterLink from './FilterLink';
+import type {ReportArtistT, ReportDataT} from './types';
+
+const DuplicateArtists = ({
+  $c,
+  canBeFiltered,
+  filtered,
+  generated,
+  items,
+  pager,
+}: ReportDataT<ReportArtistT>) => {
+  let currentKey = '';
+  let lastKey = '';
+
+  return (
+    <Layout fullWidth title={l('Possibly duplicate artists')}>
+      <h1>{l('Possibly duplicate artists')}</h1>
+
+      <ul>
+        <li>{l('This report aims to identify artists with very similar names. If \
+              two artists are actually the same, please merge them (remember to \
+              {how_to_write_edit_notes|write an edit note} and give your proof). \
+              If they\'re different, add {disambiguation_comment|disambiguation \
+              comments} to them (and once a group of similarly named artists have \
+              disambiguation comments, they will stop appearing here).',
+        {
+          disambiguation_comment: '/doc/Disambiguation_Comment',
+          how_to_write_edit_notes: '/doc/How_to_Write_Edit_Notes',
+        })}
+        </li>
+        <li>{l('Total artists found: {count}', {count: pager.total_entries})}</li>
```

Comment:
The old template uses "Total duplicate groups" here, which I guess is intended since there are actually more artists than `count`? (Multiple being in each group.)